src/main/php/web/frontend/Delegate.class.php

```diff
@@ -47,30 +47,30 @@ public function name() {
    * Returns a map of named sources to read arguments from request. Lazily
    * initialized on first use.
    *
-   * @return [:(function(web.Request, string): var)]
+   * @return [:web.frontend.Parameter]
```

Comment:
This *Parameter* class is invokeable, thus old code relying on being to use `$param($request, $name)` to read the parameter's value will continue to work!